test/clj/rems/test_browser.clj

```diff
@@ -549,6 +549,78 @@
                  last
                  (dissoc :event/id :event/time :event/attachments :event/actor-attributes)))))))
 
+(deftest test-invite-decider
+  (testing "create test data"
+    (btu/context-assoc! :form-id (test-helpers/create-form! {:form/fields [{:field/title {:en "description" :fi "kuvaus" :sv "rubrik"}
+                                                                            :field/optional false
+                                                                            :field/type :description}]}))
+    (btu/context-assoc! :catalogue-id (test-helpers/create-catalogue-item! {:form-id (btu/context-get :form-id)}))
+    (btu/context-assoc! :application-id (test-helpers/create-draft! "alice"
+                                                                    [(btu/context-get :catalogue-id)]
+                                                                    "test-invite-decider"))
+    (test-helpers/submit-application (btu/context-get :application-id) "alice"))
+  (btu/with-postmortem
+    (testing "handler invites reviewer"
+      (login-as "developer")
+      (go-to-application (btu/context-get :application-id))
+      (btu/wait-visible {:tag :h1 :fn/has-text "test-invite-decider"})
+
+      (btu/scroll-and-click :request-decision-dropdown)
+      (Thread/sleep 100) ;; need to wait for dropdown to expand
```

Comment:
Why is this wait needed if you have `wait-visible` next?

Reply:
It looked like the wait-visible on its own wasn't enough, but I guess the wait-visible is redundant now. I'll double check what's needed.

Reply:
removed